annlite/executor.py

```diff
@@ -111,6 +151,11 @@ def update(
 
             - 'traversal_paths' (str): traversal path for the docs
         """
+        if len(self._data_buffer) > 0:
+            self.logger.info(
+                'updating operation is not allowed when len(task queue) > 0'
+            )
+            return
```

Comment:
raised an exception to reject the update request https://docs.jina.ai/fundamentals/flow/when-things-go-wrong/#executor-errors